vulcanai/models/basenetwork.py

```diff
@@ -784,6 +775,9 @@ def _train_epoch(self, train_loader, retain_graph):
         train_accuracy = train_accuracy_accumulator * \
             train_loader.batch_size / len(train_loader.dataset)
 
+        # returns the network to evaluation state
+        self.eval()
```

Comment:
shouldn't this be at the end of the entire training loop instead of between epochs?

Reply:
well, there's validation within the training  loop and I wanted .eval() to be the default. We don't need .train() anywhere except here do we?

Reply:
oh I see what you mean. let's add that little tidbit of insight to the comment to make it clear why it's here and not elsewhere. thanks!

Reply:
done